prclz/reblock/utils.py

```diff
@@ -0,0 +1,125 @@
+import argparse
+import os
+import sys
+import time
+import typing
+from itertools import chain, combinations, permutations
+from pathlib import Path
+from typing import Dict, Tuple, Union
+from logging import warning
+
+import geopandas as gpd
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+from shapely.geometry import (LineString, MultiLineString, MultiPolygon, Point,
+                              Polygon)
+from shapely.ops import cascaded_union
+from shapely.wkt import loads
+
+from ..etl.commons import build_data_dir
+from .planar_graph import PlanarGraph
+
+def to_gdf(shape):
+    if isinstance(shape, list):
+        return gpd.GeoDataFrame.from_dict({'geometry':shape})
+    else:
+        return gpd.GeoDataFrame.from_dict({'geometry':[shape]})
+
+def viz_reblock(parcel_geom: MultiLineString,
+                bldg_geoms: MultiPolygon,
+                block_geom: Polygon,
+                new_steiner: MultiLineString,
+                exist_steiner: MultiLineString,
+                ):
+    parcel_geom = to_gdf(parcel_geom)
+    bldg_geoms = to_gdf(bldg_geoms)
+    block_geom = to_gdf(block_geom)
+    new_steiner = to_gdf(new_steiner)
+    exist_steiner = to_gdf(exist_steiner)
+    
+    ax = block_geom.plot(edgecolor='black', alpha=0.1)
+    ax = parcel_geom.plot(edgecolor='black', alpha=0.5, ax=ax)
+    ax = bldg_geoms.plot(color='black', ax=ax)
+    ax = new_steiner.plot(color='red', ax=ax)
+    ax = exist_steiner.plot(color='green', ax=ax)
+    return ax
+
+def debug_load():
```

Comment:
these hardcoded paths worry me - can we have this function take in a directory or set of paths?

Reply:
oh this is just something I'm using to test -- it'll be removed once I'm done testing

Reply:
i don't think WIP stuff should be committed to master - it should live on a separate feature branch